hpilo.py

```diff
@@ -856,6 +856,11 @@ def delete_user(self, user_login):
         """Delete the specified user from the ilo"""
         return self._control_tag('USER_INFO', 'DELETE_USER', attrib={'USER_LOGIN': user_login})
 
+    def delete_license(self):
```

Comment:
That function should be called `deactivate_license` to match the xml tag.

Reply:
Requested changes have been made